Gemfile

```diff
@@ -0,0 +1,6 @@
+source "https://rubygems.org"
+
+git_source(:github) { |repo_name| "https://github.com/#{repo_name}" }
```

Comment:
wip?

Reply:
Nope. This is a standard Gemfile for a Gem.